jnigen/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.4.0
```

Comment:
pubspec.yaml should also be 0.4.0

If we're not yet ready to release 0.4.0, it should be 0.4.0-dev. Then the first (and only) commit that marks it 0.4.0 should be tagged v0.4.0 and be released as 0.4.0. And we can then automate releasing with a GitHub bot (as here: https://github.com/dart-lang/tools/actions/workflows/publish.yaml)